CHANGELOG.md

```diff
@@ -1,29 +1,30 @@
 # Changelog
 
-## 1.2.0 (Unreleased)
+## 1.2.0
 
 ### Improvements
-* Now uses [OpenSSL 1.1.1.d](https://www.openssl.org/source/openssl-1.1.1d.tar.gz)
+* Now uses [OpenSSL 1.1.1.d](https://www.openssl.org/source/openssl-1.1.1d.tar.gz). [PR #60](https://github.com/corretto/amazon-corretto-crypto-provider/pull/60)
 
 ### Patches
-* Detects stuck AMD Ryzen RDRAND and correctly treats as an error
+* Detects stuck AMD Ryzen RDRAND and correctly treats as an error [PR #67](https://github.com/corretto/amazon-corretto-crypto-provider/pull/67)
 * `KeyPairGenerator` for "EC" keys when initialized with a `int` always uses "secp*r1" curves now.
```

Comment:
When initialized with an `int`, `KeyPairGenerator` for "EC" keys  now always uses "secp*r1" curves.